mstm/__init__.py

```diff
@@ -40,9 +40,11 @@
 import os
 import shutil
 import numpy as np
+from scipy import interpolate
+from scipy import integrate
 
 # change to match the filename of your executable
-MSTM_EXE = 'mstm.exe'
+MSTM_EXE = 'mstm_ubuntu.exe'
```

Comment:
I think we should just assume that the mstm executable is in the user's path.  Then the user doesn't have to deal with changing this line. Also I don't think it's necessary to copy it to the tmp directory.  So just assume it can be invoked by calling "mstm".